locale/en/docs/index.md

```diff
@@ -23,20 +23,7 @@ This documentation describes the built-in modules provided by Node.js. It does n
 
 ### Looking for API docs of previous releases?
 
-* [Node.js 15.x](https://nodejs.org/docs/latest-v15.x/api/)
-* [Node.js 13.x](https://nodejs.org/docs/latest-v13.x/api/)
-* [Node.js 12.x](https://nodejs.org/docs/latest-v12.x/api/)
-* [Node.js 11.x](https://nodejs.org/docs/latest-v11.x/api/)
-* [Node.js 10.x](https://nodejs.org/docs/latest-v10.x/api/)
-* [Node.js 9.x](https://nodejs.org/docs/latest-v9.x/api/)
-* [Node.js 8.x](https://nodejs.org/docs/latest-v8.x/api/)
-* [Node.js 7.x](https://nodejs.org/docs/latest-v7.x/api/)
-* [Node.js 6.x](https://nodejs.org/docs/latest-v6.x/api/)
-* [Node.js 5.x](https://nodejs.org/docs/latest-v5.x/api/)
-* [Node.js 4.x](https://nodejs.org/docs/latest-v4.x/api/)
-* [Node.js 0.12.x](https://nodejs.org/docs/latest-v0.12.x/api/)
-* [Node.js 0.10.x](https://nodejs.org/docs/latest-v0.10.x/api/)
-* [All versions](https://nodejs.org/docs/)
+{#var nodeVersionLinks}* [All versions](https://nodejs.org/docs/)
```

Comment:
This is purposely written like this, because we need to make a whole ```<ul>....</ul>```. If you begin with a new line for [All Versions], it will generate ```<li><p>...<p></li>```, this isn't what it used to be.